src/ngraph/runtime/intelgpu/unit_test.manifest

```diff
@@ -19,6 +19,7 @@ batch_norm_one_output
 batch_norm_three_outputs
 dequantize
 dequantize_axes
+dequantize_cpu
```

Comment:
This test is not a generalized backend unit test (not NGRAPH_TEST) so it does not need to be in the manifest